python/nano/src/bigdl/nano/pytorch/lightning.py

```diff
@@ -53,10 +53,17 @@ def __init__(self, model: nn.Module, loss: _Loss = None, optimizer: Optimizer =
         self.optimizer = optimizer
         self.scheduler = scheduler
         self.metrics = metrics
+        self._forward_args_ = None
 
     @property
     def forward_args(self):  # noqa
-        return inspect.getfullargspec(self.model.forward).args[1:]
+        if self._forward_args_ is None:
+            self._forward_args_ = inspect.getfullargspec(self.model.forward).args[1:]
+        return self._forward_args_
+
+    @forward_args.setter
+    def forward_args(self, forward_args):
+        self._forward_args_ = forward_args
```

Comment:
What's this for?

Reply:
To make forward_args editable, and for PytorchOpenVINOModel and similar models, I can pass model.forward_args to lightning.forward_args.